extensions/git/src/timelineProvider.ts

```diff
@@ -240,6 +240,6 @@ export class GitTimelineProvider implements TimelineProvider {
 
 	@debounce(500)
 	private fireChanged() {
-		this._onDidChange.fire(undefined);
+		this._onDidChange.fire(undefined as any);
```

Comment:
A better fix IMO would be to update the event to be this instead (which is what the VS Code repo has done)

`private _onDidChange = new EventEmitter<TimelineChangeEvent | undefined>();`